src/Control/FunFlow/ContentHashable.hs

```diff
@@ -332,25 +335,25 @@ instance (GContentHashable a, GContentHashable b) => GContentHashable (a :+: b)
 --   gContentHashUpdate ctx x = _ (unComp1 x)
 
 
-instance ContentHashable (Path.Path Path.Abs Path.File) where
+instance ContentHashable IO (Path.Path Path.Abs Path.File) where
```

Comment:
I think these could work in any `Monad m`. They only hash the string representation of the path. For hashing the actual file contents we have `FileContent` and `DirectoryContent.`